src/components/challenges/active.tsx

```diff
@@ -33,38 +33,23 @@ type ActiveProps = {
 const Active: FC<ActiveProps> = ({ content }) => {
   const [isUserSubscribed, setUserSubscribed] = useState(content.is_user_subscribed);
 
-  const subscribeEventHandler = async () => {
-    setUserSubscribed(1);
-  };
-
   return (
-    <div className={classNames.boxContent}>
-      <p className={classNames.heading}>{content.title}</p>
-      <Details text="Level" value={content.level} />
-      <Details text="Challenge Started" value={content.start_date} />
-      <Details text="Challenge Ends" value={content.end_date} />
-      <div className={classNames.participants}>
-        <Details
-          text="Active Participants"
-          value={content.participants.length}
-        />
-        <Participants participants={content.participants} />
-      </div>
-      {
-        !isUserSubscribed && (
-          <p className={classNames.activeBtn}>
-            <button
-              onClick={subscribeEventHandler}
-              onKeyDown={subscribeEventHandler}
-              tabIndex={0}
-              type="button"
-            >
-              I will do this
-            </button>
-          </p>
-        )
+    <Card
+      title={{ text: content.title }}
+      data={details(content)}
+      participants={participantsDetails(content)}
+      button={
+        {
+          text: 'I will do this',
+          onClick: async () => {
```

Comment:
why `async`?

Reply:
Removed async as it was unnecessary.

https://github.com/Real-Dev-Squad/website-status/pull/101/commits/a0be4dda9052434e25c95854ca958518d8a29e6d